MIGRATION_GUIDE.md

```diff
@@ -11,6 +11,8 @@ Looking to upgrade from Sentry SDK 2.x to 3.x? Here's a comprehensive list of wh
 
 ### Removed
 
+- When setting span status, the HTTP status code is no longer automatically added as a tag.
```

Comment:
very good point you raise here,
let's make sure to have a follow up with relay to make sure we agree on the 'right way' going forward, both the SDK side and the relay side has a whole bunch of weird logic for getting the status from various sources.

Let's simplify if possible. cc @AbhiPrasad

Reply:
ah ok @antonpirker said the same